squad_client/core/models.py

```diff
@@ -569,6 +569,40 @@ def watch(self):
             backend=self.backend,
             testjob_id=self.job_id,)
 
+    __testrun_obj__ = None
+
+    @property
+    def testrun_obj(self):
```

Comment:
I use the name of testrun_obj here to distinguish with the testrun attribute, which is the url of the api.
it seems not work to remove that testrun attribute and rename this to testrun, as the api url will be used to get the various value for the TestRun object.
Please give suggestion if you think we should have another name for this.